src/python/pants/backend/python/lint/flake8/rules_integration_test.py

```diff
@@ -120,48 +116,6 @@ def test_multiple_targets(rule_runner: PythonRuleRunner) -> None:
     assert "bad.py:1:1: F401" in result[0].stdout
 
 
-@skip_unless_python37_and_python39_present
```

Comment:
I was unable to figure out a way to get flake8 to spit out `E999` for Python3.8 code that would be valid in 3.9.  I remain skeptical of the sustainability of this pattern.

```
$ python --version
Python 3.8.19
$ cat f.py 
y = {} | {}
$ flake8 f.py 
$ echo $?
0
```